docs/core/tools/dotnet-store.md

```diff
@@ -0,0 +1,78 @@
+---
+title: dotnet-store command | .NET Core SDK
+description: The dotnet-store Stores the specified assemblies in the runtime package store.
+keywords: dotnet-store, dotnet-publish, CLI, CLI command, .NET Core
+author: bleroy
+ms.author: beleroy
+ms.date: 4/13/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: 1e8e4122-8110-4b48-afce-afffb6737776
+---
+
+#dotnet-store
+
+## Name
+
+`dotnet-store` - Stores the specified assemblies in the [runtime package store](../deploying/runtime-package-store.md).
+
+## Synopsis
+
+`dotnet store [--manifest] [--framework-version] [--output] [--working-dir] [--preserve-working-dir] [--skip-optimization] [-v|--verbosity] [-h|--help]`
```

Comment:
Code is always the best reference - it can't lie. :)

Example of the file that gets passed into `dotnet store --manifest`: https://github.com/dotnet/sdk/blob/0aa4c2101b1b01c81cf4c0c584a4354e40572bae/TestAssets/TestProjects/TargetManifests/NewtonsoftFilterProfile.xml

Although we never really expect users to build this up themselves (we expect them to take the file that is produced by `dotnet store`), here is an example of the file that gets passed into `dotnet publish --manifest`: https://github.com/dotnet/sdk/blob/0aa4c2101b1b01c81cf4c0c584a4354e40572bae/TestAssets/TestProjects/StoreManifests/NewtonsoftFilterProfile.xml